tendrl/gluster_integration/objects/node_context/__init__.py

```diff
@@ -37,7 +37,7 @@ def _create_node_id(self, node_id=None):
         with open(local_node_context, 'wb+') as f:
```

Comment:
Don't we plan to maintain node context specific to cluster anymore? At the moment I am aware of consequences of removing this.